docs/managing-minishift.md

```diff
@@ -217,6 +231,36 @@ authenticated users. The following procedure describes how to mount this folder.
 - If your Windows account is linked to a Microsoft account, you must use the full Microsoft account email address to
   authenticate, for example `jpillow@amigas.us`. This ensures that the domain value that contains the computer name is provided.
 
+
+<a name="sshfs"></a>
+#### SSHFS
+
+You can use SSHFS based host folder mounts, when you have a SSH daemon running on your host. For Linux
+and OS X this prerequisite is most of the time already met. Most Linux distributions have a SSH daemon
+installed per default. If not, follow the instructions for your specific distribution to install one.
+OS X also has a built-in SSH server. To use it, make sure _Remote Login_ is enabled under
+_System Preferences > Sharing_. On Windows you should consider using a [CIFS based](#cifs)
+mount first, but if you want to try SSHFS, you can install [OpenSSH for Windows](https://winscp.net/eng/docs/guide_windows_openssh_server).
+
+1. Similar to the [CIFS]((#cifs)) example, you need the IP address from which your host is reachable
+from within the VM. Run `ifconfig` (or `Get-NetIPAddress` on Windows) to determine the local IP
```

Comment:
you need the local IP address so that your host is reachable from within the VM?